src/main/java/org/jfree/chart/date/SerialDate.java

```diff
@@ -538,14 +552,17 @@ public static SerialDate addDays(int days, SerialDate base) {
      *
      * @return a new date.
      */
+    @SuppressWarnings({"index", "value"}) // True positive, fixed upstream bug
```

Comment:
Reference an issue tracker bug in upstream JFreeChart repository.
The same comment applies to other comments of this form.